frontend/src/libs/components/button/button.tsx

```diff
@@ -1,11 +1,32 @@
+import { faChevronDown, faPlus } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import styles from './button.module.scss';
+
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  size?: 'sm' | 'md';
+  variant?: 'contained' | 'outlined' | 'text';
+  disabled?: boolean;
 };
 
 const Button: React.FC<Properties> = ({
   type = 'button',
+  size = 'md',
+  variant = 'contained',
   label,
-}: Properties) => <button type={type}>{label}</button>;
+  disabled = false,
+}: Properties) => (
+  <button
+    className={[styles.btn, styles[size], styles[variant]].join(' ')}
```

Comment:
we will use clsx

Reply:
I remember we discussed this. I wanted to change when clsx will be added to the project. Or should I install it myself?

Reply:
created a separate task for clsx library. Let's wait for it, I think it will be implemented soon, and then u can reuse it
https://github.com/BinaryStudioAcademy/bsa-2023-towhub/issues/55

Reply:
Got it, thanks.